HexCtrl/src/CHexCtrl.cpp

```diff
@@ -385,6 +385,10 @@ bool CHexCtrl::Create(const HEXCREATE& hcs)
 	m_pDlgSearch->Create(IDD_HEXCTRL_SEARCH, this, this);
 	m_pDlgGoTo->Create(IDD_HEXCTRL_GOTO, this, this);
 	m_pBookmarks->Attach(this);
+
+	//Determine current user locale specific date format. Default to UK/European if unable to determine
+	m_dwDateFormat = GetUserDefaultDateFormat();
+	

```

Comment:
What does it do exactly?
It assigns `m_dwDateFormat` to itself. 

Actually it must be `SetDateFormat(-1);`

Reply:
Fixed.